packages/financial-templates-lib/src/helpers/GasEstimator.js

```diff
@@ -96,21 +119,33 @@ class GasEstimator {
       });
 
       // Try backup API.
-      try {
-        const responseBackup = await fetch(backupUrl);
-        const jsonBackup = await responseBackup.json();
-        if (jsonBackup.result && jsonBackup.result.ProposeGasPrice) {
-          return jsonBackup.result.ProposeGasPrice;
-        } else {
-          throw new Error("Etherscan API: bad json response");
+      if (backupUrl) {
+        try {
+          const responseBackup = await fetch(backupUrl);
+          const jsonBackup = await responseBackup.json();
+          // Etherscan API limits 1 request every 3 seconds without passing in an API key. Expected response structure:
+          // {
+          //   "status": "1",
+          //   "message": "OK-Missing/Invalid API Key, rate limit of 1/3sec applied",
+          //   "result": {
+          //       "LastBlock": "10330323",
+          //       "SafeGasPrice": "30",
+          //       "ProposeGasPrice": "41"
+          //   }
+          // }
+          if (jsonBackup.result && jsonBackup.result.ProposeGasPrice) {
+            return jsonBackup.result.ProposeGasPrice;
+          } else {
+            throw new Error("Etherscan API: bad json response");
```

Comment:
this comment is probs wrong. it's the catch from a throw. it's not strictly etherscan API, right? it's just the backup one. (I know this was here before your PR).